packages/zipkin/src/internalRepresentations.js

```diff
@@ -15,6 +15,16 @@ Endpoint.prototype.toThrift = function toThrift() {
     port: this.port
   });
 };
+Endpoint.prototype.toJSON = function toJSON() {
+  return {
+    ipv4: this.formatIPv4(this.host),
+    port: this.port
+  };
+};
+Endpoint.prototype.formatIPv4 = function formatIPv4(host) {
```

Comment:
This function doesn't need to be on Endpoint.prototype, it can live on its own. (It doesn't need access to `this`)